app/assets/stylesheets/application.scss

```diff
@@ -14,22 +14,31 @@
  *= require_self
  */
 
+$input-placeholder-color: hsl(0, 0%, 86%); //has-text-grey-lighter
```

Comment:
These changes are in another PR, but I pulled them in here bc I'm building upon the styling to make the public-facing pages (and the internal ones, but...).